tsconfig.json

```diff
@@ -10,6 +10,7 @@
     "dist",
     "libs",
     "bower",
-    "tsd"
+    "tsd",
```

Comment:
This is for ignoring tsd file, which might not yet be removed because it's generated. This is just so building will not throw errors.